### ISSUE TITLE
default options don't match original game behavior

### ISSUE THREAD

**Issue by zdechlak** [forkserf/forkserf#330]
![image](https://user-images.githubusercontent.com/2237225/218320858-69c2fd17-435e-4aea-a3bf-7e24d01bea27.png)

this is probably not a standard value for reset button / I'm not sure

**Comment by tlongstretch** [comment]
Yes there should probably be two separate sets of defaults, one for "original game behavior" and another for "recommended for most players"
